### PR TITLE
Clean up the whoopsies that snuck into prerelease 3782

### PR DESCRIPTION
**Pull request by Vogel612**

### CODE REVIEW

File: Rubberduck.Core/UI/Command/InspectionResultsCommand.cs [hunk 1]
```diff
@@ -9,9 +10,9 @@ namespace Rubberduck.UI.Command
     [ComVisible(false)]
     public class InspectionResultsCommand : CommandBase
     {
-        private readonly IDockablePresenter _presenter;
+        private readonly InspectionResultsDockablePresenter _presenter;
```
Comment (MDoerner) [inline Rubberduck.Core/UI/Command/InspectionResultsCommand.cs]: Why have the concrete type here?

Comment (retailcoder) [reply Rubberduck.Core/UI/Command/InspectionResultsCommand.cs]: This was shortly discussed in chat yesterday ([permalink](https://chat.stackexchange.com/transcript/message/46785598#46785598)) - bottom line, there's definitely a design problem with these, but in the current state of things we don't have tests at the command level (right?), so the decoupling is kind of artificial and the interfaces are kind of violating LSP anyway - injecting `BarDockablePresenter` in `FooCommand` doesn't make any sense, the coupling is fine for now I think, until we come up with a better design.

**Comment by retailcoder** [comment]
I'm going to merge this PR given the urgency of the fix, but I'd like to echo @MDoerner's comments about the CW configuration, they should be addressed in either a small-ish tech-debt PR, or a (rather large, admittedly) dockable-stuff/command mapping redesign PR.

File: Rubberduck.Main/Root/RubberduckIoCInstaller.cs [hunk 2]
```diff
@@ -712,17 +667,11 @@ private void RegisterDockableUserControls(IWindsorContainer container)
...
-            container.Register(Component.For<IDockablePresenter>()
-                .ImplementedBy<CodeExplorerDockablePresenter>()
-                .LifestyleSingleton());
-            container.Register(Component.For<IDockablePresenter>()
-                .ImplementedBy<ToDoExplorerDockablePresenter>()
+            container.Register(Classes.FromAssemblyContaining<IDockablePresenter>()
+                .IncludeNonPublicTypes()
+                .BasedOn<IDockablePresenter>()
+                .WithServiceSelf()
+                .WithServices(new[] { typeof(IDockablePresenter) })
```
[the remark this answers is not in the document]
Comment (Vogel612) [reply Rubberduck.Main/Root/RubberduckIoCInstaller.cs]: This is just to play things safe and bandaid stuff before properly fixing, see #4379 which is basically exactly what you've proposed, just not as hotfix

**Pull request merged**